.zuul.d/collect-logs.yml

```diff
@@ -15,6 +15,15 @@
             chdir: "{{ ansible_user_dir }}/zuul-output/logs"
             cmd: cp /tmp/report.html .
 
+    - name: Get some env related data
+      ansible.builtin.shell: |
+        rpm -qa | sort > {{ ansible_user_dir }}/zuul-output/logs/installed-pkgs.log;
+        rpm -ql python3-libvirt > {{ ansible_user_dir }}/zuul-output/logs/python3-libvirt.log;
```

Comment:
Any specific reason to store python3-libvirt rpm version in a separate file?

Reply:
Ah, you want to list all the files installed by rpm https://logserver.rdoproject.org/83/83/0bd0f6a691e4a3357daae26d6e62f6d9597696ca/github-check/end-to-end/7bf5867/python3-libvirt.log.Do we still need it.

Reply:
old debugging - removed.